README.md

```diff
@@ -124,112 +124,24 @@ const NotFound = () => <h3>404</h3>
-```js
-// reducers/title.js
-const DEFAULT = 'RFR demo'
-
-export default (state = DEFAULT, action = {}) => {
-  switch (action.type) {
-    case 'HOME':
-      return DEFAULT
-    case 'USER':
-      return `${DEFAULT} - user ${action.payload.id}`
-    default:
-      return state
-  }
-}
-```
-
-```js
-// reducers/index.js
-export { default as title } from './title'
-```
-
-```diff
-// configureStore.js
-+ import * as reducers from './reducers';
-
-- const rootReducer = combineReducers({ page, title, location: reducer })
-+ const rootReducer = combineReducers({ ...reducers, page, title, location: reducer })
-```
-
-#### Embed SEO-friendly links
-`yarn install redux-first-router-link`
-
-```js
-import Link from 'redux-first-router-link'
-
-<Link to="/user/123">User 123</Link>
-// Recommended approach - URLs can be changed by only modifying routesMap.js
-<Link to={{ type: 'USER', payload: { id: 456 } }}>User 456</Link>
-
-// Same as above but without SEO benefits
-const mapDispatchToProps = dispatch => ({
-  onClick: id => dispatch({ type: 'USER', payload: { id } })
-})
-```
-#### Style active links
-The link components are in a separate package, see above. `<NavLink />` contains props such as `activeStyle` and `activeClassName`.
-**TODO**
-
-#### Dispatch thunk on route change
-**TODO**
-
-#### Perform redirects
-Redirect to another route based on `payload` and `state`.
-
-**TODO**
-
-#### Use hash-based routes/history
-**TODO**: Introduce options.js and `createHashHistory` from `history`/`rudy-history`.
-See the [migration instructions](https://github.com/faceyspacey/redux-first-router/tree/master/docs/migration.md).
-
-#### Restore scroll position
-**TODO**
-
-#### Handle query strings
-**TODO**
-
-#### Pre- & post-process route changes
-**TODO**
-
-#### Enable code-splitting
-**TODO**
-
-#### Use Redux Devtools to debug route changes
-```diff
-// configureStore.js
--  const enhancers = compose(enhancer, middlewares)
-+  const composeEnhancers =
-+    typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
-+      ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({})
-+      : compose
-+  const enhancers = composeEnhancers(enhancer, middlewares)
-```
-
-Dispatch `{ type: 'USER', payload: { id: 13 } }` and see the route change.
-Then use the _Inspector_ to time travel back to `HOME`.
-
-## Server-side rendering
-
-**TODO**: Usage together with
-`react-universal-component`, `babel-plugin-universal-import`, `webpack-flush-chunks`.
+- *Dispatching thunks on route changes*
+- *Performing redirects bases on `state` and `payload`.*
+- *Use hash-based routes/history (*see the [migration instructions](./docs/migration.md)*)*
+- *Restoring scroll position*
+- *Handling optional URL fragments and query strings*
+- *Route change pre- & post-processing*
+- *Code-splitting*
+- *Server-side rendering*
+- *Usage together with `react-universal-component`, `babel-plugin-universal-import`, `webpack-flush-chunks`.*
```

Comment:
The last three are nicely demonstrated in `redux-first-router-link` - perhaps its worth adding a link to that?

Reply:
Sounds good, will check those out as the next step after this. Would be awesome to finish that list and then get started on ./docs!

Reply:
@hedgepigdaniel I added links to redux-first-router-link for the "links" examples. But I didn't find any examples of how to do the last three items above (SSR & universal etc), were you thinking of another repo?